lldb-eval/parser.cc

```diff
@@ -252,6 +252,13 @@ static ExprResult UsualUnaryConversions(lldb::SBTarget target,
 
   // TODO(werat): Promote bitfields: e.g. uint32_t:10 -> int (not unsigned int).
 
+  if (result_type.IsArrayType()) {
```

Comment:
Nit: can you update the comment few lines above saying that this only includes the integral promotion?